DigitalLearningSolutions.Web/Styles/shared/searchableElements/common.scss

```diff
@@ -107,3 +95,36 @@ $button-hover-colour: #4c6272;
     background-color: $button-hover-colour;
   }
 }
+
+.search-sort-flex-container {

```

Comment:
We might want to think through namings a bit - everything being called a "flex-container" makes it quite difficult to follow. I'd probably remove "flex" from the name entirely, such that the name doesn't need to be changed if the display value gets changed in the CSS.